src/main/java/lotto/service/LottoServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package lotto.service;
+
+import lotto.model.Lotto;
+import lotto.model.LottoResult;
+import lotto.model.WinningNumbers;
+import lotto.repository.LottoRepository;
+import lotto.view.OutputView;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoServiceImpl implements LottoService {
+
+    private final LottoRepository lottoRepository;
+    private final LottoResult lottoResult;
+    private static final int LOOTO_AMOUNT = 1000;
+
+    public LottoServiceImpl(LottoRepository lottoRepository) {
+        this.lottoRepository = lottoRepository;
+        this.lottoResult = new LottoResult();
+    }
+
+    @Override
+    public void purchaseLottos(int amount) {
+        Lotto.validatePurchaseAmount(amount);
+
+        int numberOfLotto = amount / LOOTO_AMOUNT;
+        IntStream.range(0, numberOfLotto).forEach(i -> {
+            Lotto lotto = Lotto.createRandomLotto(); // Lotto도메인에서 팩토리 메소드 호출
+            saveLottoNums(lotto);
```

Comment:
분리하지 않고 합쳐도 괜찮을 것 같습니다.
```suggestion
            saveLottoNums(Lotto.createRandomLotto());
```

주석 사용은 최대한 자제해주세요. 코드만 보고 읽히는 것이 좋은 코드입니다.